modules/packages/ZMQ.chpl

```diff
@@ -891,7 +891,7 @@ module ZMQ {
         // 
         // Note: the string factory below can throw DecodeError
         var copy = if isString(T) then createStringWithNewBuffer(x=data)
-                                  else createBytesWithNewBuffer(x=data);
+                                  else parallelCreateBytesWithNewBuffer(data.localize().buff, length=data.size);        
```

Comment:
I pay attention to keeping lines below 80 chars, but I know that we aren't very consistent with doing that across our repo.